Description: fix compile error on arch=x32
 On the x32 architecture time_t is a "long long int". By
 typecasting time_t to "long long", this should work correctly
 for all architectures.
Author: John Ogness <john.ogness@linutronix.de>
Last-Update: 2016-12-02

--- minicoredumper-2.0.0.orig/src/minicoredumper/corestripper.c
+++ minicoredumper-2.0.0/src/minicoredumper/corestripper.c
@@ -1241,7 +1241,7 @@ static int dump_compressed_tar(struct du
 	snprintf(hdr.mode, sizeof(hdr.mode), "%07o", 0644);
 	snprintf(hdr.uid, sizeof(hdr.uid), "%07o", 0);
 	snprintf(hdr.gid, sizeof(hdr.gid), "%07o", 0);
-	snprintf(hdr.mtime, sizeof(hdr.mtime), "%011lo", time(NULL));
+	snprintf(hdr.mtime, sizeof(hdr.mtime), "%011llo", (long long)time(NULL));
 	memset(hdr.checksum, ' ', sizeof(hdr.checksum));
 	hdr.type = 'S';
 	memcpy(hdr.magic, "ustar ", 6);
